### PR TITLE
DHBs data list extract to its own component

### PR DESCRIPTION
**Pull request by andycarrell**
Pretty minor, but allows us to simplify globals `scss` 💃

### CODE REVIEW

File: app/components/DhbsVaccineDoseDataList/DhbsVaccineDoseDataList.module.scss [hunk 1]
```diff
@@ -0,0 +1,22 @@
+@import "../../styles/spacing.scss";
+
+.grid {
```
Comment (finnclark) [inline app/components/DhbsVaccineDoseDataList/DhbsVaccineDoseDataList.module.scss:3]: Just something a little more specific could be good 🤷 

```suggestion
.card-grid {
```

File: app/components/DhbsVaccineDoseDataList/DhbsVaccineDoseDataList.module.scss [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  display: grid;
+  grid-template-columns: repeat(12, minmax(0, 1fr));
+  gap: $space-l;
+}
+
+.item {
```
Comment (finnclark) [inline app/components/DhbsVaccineDoseDataList/DhbsVaccineDoseDataList.module.scss:9]: 🤷 👍 👎 

```suggestion
.card {
```

File: app/components/DhbsVaccineDoseDataList/DhbsVaccineDoseDataList.tsx [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+import { DhbVaccineDoseDataForIndexPage } from "../../types/IndexPageProps";
+import dhbDisplayName from "../../utilities/dhbDisplayName";
+
+import { DosesDescriptionList } from "../DosesDescriptionList";
+import { Progress } from "../Progress";
+
+import styles from "./DhbsVaccineDoseDataList.module.scss";
+
+type DhbsVaccineDoseDataListProps = {
+  dhbsVaccineDoseData: DhbVaccineDoseDataForIndexPage[];
```
Comment (jishaal) [inline app/components/DhbsVaccineDoseDataList/DhbsVaccineDoseDataList.tsx:12]: Feels odd with the type naming here, but I think I can tidy this up in my PR next

File: app/components/DhbsVaccineDoseDataList/DhbsVaccineDoseDataList.tsx [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+};
+
+export const DhbsVaccineDoseDataList = ({
+  dhbsVaccineDoseData,
+}: DhbsVaccineDoseDataListProps) => (
+  <div className={styles["grid"]}>
```
Comment (finnclark) [inline app/components/DhbsVaccineDoseDataList/DhbsVaccineDoseDataList.tsx:18]:
```suggestion
  <div className={styles["card-grid"]}>
```

**Review by jishaal — APPROVED** [review]

File: app/components/DhbsVaccineDoseDataList/DhbsVaccineDoseDataList.tsx [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+    {dhbsVaccineDoseData.map((dhbVaccineDoseData) => (
+      <div
+        key={dhbVaccineDoseData.dhbName}
+        className={cx(styles["item"], "padding-l")}
```
Comment (finnclark) [inline app/components/DhbsVaccineDoseDataList/DhbsVaccineDoseDataList.tsx:22]:
```suggestion
        className={cx(styles["card"], "padding-l")}
```

**Review by finnclark — APPROVED** [review]
So much tidier! 🙌  Just a couple naming suggestions

File: app/components/DhbsVaccineDoseDataList/DhbsVaccineDoseDataList.tsx (hunk 3 above)
Comment (andycarrell) [reply app/components/DhbsVaccineDoseDataList/DhbsVaccineDoseDataList.tsx:12]: Yeh I would say if we can't pass data through like this it's not a great type name - keen if you can to rename in another pr 🙏

**Comment by andycarrell** [comment]
@finnclark - will rename in the next PR for ya!

**Pull request merged**